Source/Streamstone/Stream.Operations.cs

```diff
@@ -31,7 +31,7 @@ public Stream Execute()
 
                 try
                 {
-                    table.Execute(insert.Prepare());
+                    table.ExecuteAsync(insert.Prepare()).Wait();
```

Comment:
Yes, I know, that is why I'm saying that we should remove all the sync code from inside SS and let people decide where and how they should block or not the framework usage...

The only reason I did that, is the fact that I didn't want to change the public APIs yet but if you guys think we should, I'll be happy to do it.